client_test.go

```diff
@@ -27,11 +27,8 @@ func eventHandler(evt interface{}) {
 }
 
 func Example() {
-	// |------------------------------------------------------------------------------------------------------|
-	// | NOTE: You must also import the appropriate DB connector, e.g. github.com/mattn/go-sqlite3 for SQLite |
-	// |------------------------------------------------------------------------------------------------------|
-
 	dbLog := waLog.Stdout("Database", "DEBUG", true)
+	// Make sure you add appropriate DB connector imports, e.g. github.com/mattn/go-sqlite3 for SQLite
```

Comment:
The comment was intentionally big to make it harder to miss, so making it smaller isn't very helpful